documentation/CAMARA-API-Event-Subscription-and-Notification-Guide.md

```diff
@@ -140,9 +140,9 @@ The following table provides `/subscriptions` attributes
-| sinkCredential | object             | Sink credential provides authorization information necessary to enable delivery of events to a target. In order to be updated in future this object is polymorphic. See detail below. To protect the notification endpoint providing sinkCredential is RECOMMENDED. <br> The sinkCredential must **not** be present in `POST` and `GET` responses.                                                                                                                                                                                                                                                                    | optional                     |
-| types          | SubscriptionEventType | Type of event subscribed. This attribute **must** be present in the `POST` request. It is required by API project to provide an enum for this attribute. The event type must follow the format: `org.camaraproject.<api-name>.<event-version>.<event-name>` - see chapter [2.3. Event versioning](#23-event-versioning) - Note: An array of types could be passed **but as of now only one value MUST be passed**. Use of multiple value will be open later at API level. | mandatory                    |
+| sink           | string             | The URL, to which event notifications shall be sent - `format: uri` should be used to require a string that is compliant with [RFC 3986](https://datatracker.ietf.org/doc/html/rfc3986). The URI-scheme shall be set according to the definition of the `protocol` value, e.g. the URI-scheme is `https` when `HTTP`is the value of the `protocol` property. The [security considerations](43#notifications-security-considerations) should be followed.                                                                                 | mandatory                    |
+| sinkCredential | object             | Sink credential provides authorization information necessary to enable delivery of events to a target. In order to be updated in future this object is polymorphic. See detail below. To protect the notification endpoint providing sinkCredential is RECOMMENDED. <br> The sinkCredential must **not** be present in `POST` and `GET` responses.                                                                                                                                                                                       | optional                     |   
+| types          | SubscriptionEventType | Type of event subscribed. This attribute **must** be present in the `POST` request. It is rquired by API project to provide an enum for this attributee. The event type must follow the format: `org.camaraproject.<api-name>.<event-version>.<event-name>` - see chapter [2.3. Event versioning](#23-event-versioning) - Note: An array of types could be passed. The decision to use multiple event types in a single subscription will be made at the API level. | mandatory                    |
```

Comment:
2 small miscues:
It is **required** by API project to provide an enum for this **attribute**